ts-sdk/whirlpool/tests/token.test.ts

```diff
@@ -600,4 +600,16 @@ describe("Token Account Creation", () => {
       owner: signer.address,
     });
   });
+
+  it("Should order mints by canonical byte order", () => {
+    const mint1 = address("BRjpCHtyQLNCo8gqRUr8jtdAj5AjPYQaoqbvcZiHok1k");
+    const mint2 = address("Jd4M8bfJG3sAkd82RsGWyEXoaBXQP7njFzBwEaCTuDa");
+    const [mintA, mintB] = orderMints(mint1, mint2);
+    assert.strictEqual(mintA, mint1);
+    assert.strictEqual(mintB, mint2);
+
+    const [mintC, mintD] = orderMints(mint2, mint1);
+    assert.strictEqual(mintC, mint1);
+    assert.strictEqual(mintD, mint2);
```

Comment:
hmm, is this test data correct ?

```
import { PoolUtil } from "@orca-so/whirlpools-sdk";
import { PublicKey } from "@solana/web3.js";

async function main() {
    const mint1 = new PublicKey("BRjpCHtyQLNCo8gqRUr8jtdAj5AjPYQaoqbvcZiHok1k");
    const mint2 = new PublicKey("Jd4M8bfJG3sAkd82RsGWyEXoaBXQP7njFzBwEaCTuDa");
    const [mintA, mintB] = PoolUtil.orderMints(mint1, mint2);
    console.log("mintA == mint1", mintA.toString() === mint1.toString());
    console.log("mintB == mint2", mintB.toString() === mint2.toString());
    console.log("mintA", mintA.toString());
    console.log("mintB", mintB.toString());
}

main();
```
shows
```
mintA == mint1 false
mintB == mint2 false
mintA Jd4M8bfJG3sAkd82RsGWyEXoaBXQP7njFzBwEaCTuDa
mintB BRjpCHtyQLNCo8gqRUr8jtdAj5AjPYQaoqbvcZiHok1k
```

[Converter](https://appdevtools.com/base58-encoder-decoder) shows

- `BRjpCHtyQLNCo8gqRUr8jtdAj5AjPYQaoqbvcZiHok1k`: `9aeb37...`
- `Jd4M8bfJG3sAkd82RsGWyEXoaBXQP7njFzBwEaCTuDa`: `0483a4...`

Reply:
It should be flipped. I misread whoever reported the issue